custom/icds_reports/management/commands/sql_scripts/nos_of_deaths.sql

```diff
@@ -9,9 +9,13 @@ FROM (
       SELECT awc_location.state_name AS state,
              awc_location.district_name AS district,
              awc_location.awc_name AS awc,
```

Comment:
they only want those 3? not supervisor or block?

Reply:
yeah right